services/common/src/components/reports/ReportDetailsForm.tsx

```diff
@@ -128,22 +140,37 @@ const ReportDetailsForm: FC<ReportDetailsFormProps> = ({
 
   const system = useSelector(getSystemFlag);
 
+  // PRR
+  const permit = useSelector(getPermitByGuid(permit_guid));
+  const dropdownPermitConditionCategoryOptions = useSelector(
+    getDropdownPermitConditionCategoryOptions
+  );
+  const selectedPermitCategory =
+    permit_condition_category_code &&
+    dropdownPermitConditionCategoryOptions.find(
+      (opt) => opt.value === permit_condition_category_code
+    );
+
+  const isCRR = report_type === "CRR";
+  const isPRR = report_type === "PRR";
+
   // minespace users are only allowed to add documents
   const mineSpaceEdit =
     system === SystemFlagEnum.ms && initialValues?.mine_report_guid && isEditMode;
 
+  useEffect(() => {
+    if (permit_guid && !permit) {
+      dispatch(fetchPermits(mineGuid));
+    }
+  });
```

Comment:
Looks like the dependency array is missing here.

Reply:
it's meant to run only when the page is loaded, not run again if the permit is changed (since it does fetchPermits)

Reply:
Sorry my comment should actually have been "Looks like an empty dependency array". I think it will always run anytime the page/component is loaded.

Reply:
Without the dependency array I think it would run on every render, but due to the if statement only fetch if there is a permit guid but no permit.  That should work I think, but could theoretically do it with `permit_guid` and `permit` in the dependency array as which might be the more recommended path.

Reply:
gasp, I did have it backwards, empty dep array it is!
https://react.dev/reference/react/useEffect#examples-dependencies

If I were to put something in there though, it would actually be the mineGuid. I am not sure it would be possible to change the mine guid without just reloading the page. But permit/permit_guid would cause extra fetches when the permit is changed in CORE.